iOS Client/Utilities/Formatters.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Formatters.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 11/14/17.
+//  Copyright © 2017 Mockingbird. All rights reserved.
+//
+
+import Foundation
+
+struct Formatters {
```

Comment:
double-check me, but now that we're encapsulating a lot of this work in the LoadDevotion struct, couldn't we remove this file all-together and do these  as private things inside of LoadedDevotion struct? 

(Looks like calendar was already moved over)

1 good use of this file might be a helper that goes from Date to String in YYYY-MM-DD format since that's what we may use for comparison in the notification delegate.

Reply:
I think in the future if we needed some formatters that were shared between a couple screens or models than we could use this file. I'm ok with having all of this in LoadedDevotion for now